src/data/markdown/docs/02 javascript api/02 k6/group- name- fn -.md

```diff
@@ -17,6 +17,19 @@ Run code inside a group. Groups are used to organize results in a test.
 | ---- | ------------------------- |
 | any  | The return value of _fn_. |
 
+<Blockquote mod="warning">
+
+Avoid using `group` with async functions or asynchronous code.
+If you do, k6 might apply tags in a way that is unreliable or unintuitive.
+
+If you start promise chains or even use `await` within `group`, some code within the group will be waited for and tagged with the proper `group` tag, but others won't be.
+
+To avoid confusion, async functions are forbidden as arguments. This still let users make and chain promises within a group, but doing so is unsupported and not recommended.
```

Comment:
```suggestion
To avoid confusion, async functions are forbidden as arguments. This still lets users make and chain promises within a group, but doing so is unsupported and not recommended.
```